Banking.Database/Account/Stored Procedures/DeleteAccountType_1.sql

```diff
@@ -0,0 +1,19 @@
+﻿CREATE PROCEDURE[Account].[DeleteAccountType]
+(
+@AccId INT,
+@AccountType VARCHAR (20)
+)
+AS
+BEGIN
+	DECLARE @AccType VARCHAR(20)
+	SET @AccType=(SELECT Type FROM [Account].[AccountType] WHERE @AccountType=Type AND Id=@AccId)
```

Comment:
DECLARE @AccType AS VARCHAR(20) = '';
SELECT @AccType= Type 
FROM [Account].[AccountType] 
WHERE @AccountType = Type AND Id = @AccId;